kge/model/kg2e.py

```diff
@@ -63,10 +63,10 @@ def __init__(
         init_for_load_only=False,
     ):
         self._init_configuration(config, configuration_key)
-        self._dim = self.get_option("entity_embedder.dim")
-        self.set_option("entity_embedder.dim", self._dim * 2, log=True)
+        self.dim = self.get_option("entity_embedder.dim")
+        self.set_option("entity_embedder.dim", self.dim * 2, log=True)
         if self.get_option("relation_embedder.dim") < 0:
-            self.set_option("relation_embedder.dim",  self._dim * 2, log=True)
+            self.set_option("relation_embedder.dim", self.dim * 2, log=True)
```

Comment:
We can load the model by using the below command: 
```
import torch
from kge.model import KgeModel
from kge.util.io import load_checkpoint

# download link for this checkpoint given under results above
checkpoint = load_checkpoint('fb15k-237-rescal.pt')
model = KgeModel.create_from(checkpoint)
```
But because of the changes in dimensions, we can't load the model. 
So I think it is better to change like the below:
```
if not init_for_load_only:
    self.set_option("entity_embedder.dim", self.dim * 2, log=True)
    if self.get_option("relation_embedder.dim") < 0:
        self.set_option("relation_embedder.dim", self.dim * 2, log=True)
```